java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -485,6 +497,31 @@ private static Map<String, Change<Xor<String, List<Info>>>> extractPackageDelta(
         }
     }
 
+    /**
+     * Wrapper object representing a "changes" element containing "installed" and "removed" elements inside:
+     * "changes: { "installed": "pkg_name": { "new": "", "old": "1.7.9" } }
+     *
+     * Temporarily here until available in a new version of salt-net-api.
```

Comment:
Nitpick: can you use this note together with "@deprecated" annotation so it's more visible?